### PR TITLE
Audio: Crossover: change to use model handler to handle configs

### PR DESCRIPTION
**Pull request by eddy1021**
This is a follow-up PR for [comment](https://github.com/thesofproject/sof/pull/5702#discussion_r855465420)

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]
@jsarha can you comment on the blob handler.

**Comment by lgirdwood** [comment]
@jsarha ping.

**Comment by jsarha** [comment]
Looks good to me. Exactly the way the blob handler should be used.

**Comment by lgirdwood** [comment]
@eddy1021 any update ?

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@jsarha can you pls review the blob model handler code.
@eddy1021 looks like a CI build failure.

**Comment by eddy1021** [comment]
> @eddy1021 looks like a CI build failure.

Not sure why it shows `Tests failed`. By clicking [`Details`](https://sof-ci.01.org/sof-pr-viewer/#/build/PR5729/build9309148), all the tests show `PASS`.

File: src/audio/crossover/crossover.c [hunk 1]
```diff
@@ -837,8 +837,6 @@ static int crossover_reset(struct comp_dev *dev)
 
 	cd->crossover_process = NULL;
 	cd->crossover_split = NULL;
-	cd->config = NULL;
-	cd->config_new = NULL;
```
Comment (jsarha) [inline src/audio/crossover/crossover.c:841]: This was wrong in more ways than one, wasn't it? Assigning configs to NULL without freeing them first leaks memory... but that is fixed here too.

**Review by jsarha — APPROVED** [review]
Looks good to me.

**Comment by lgirdwood** [comment]
@wszypelt all green, but status is showing as failed ? good to merge ?

**Comment by wszypelt** [comment]
@lgirdwood  good to merg

**Pull request merged**